### PR TITLE
`stdout` will be printed if no option for output is specified

### LINKED ISSUE #80: if no option for output is specified, print to stdout
How about changing default behavior and print output to stdout by default?

if --output-dir is not set, print to stdout.
if --outdir-dir is set, save files to directory.

### PR DESCRIPTION
**Pull request by surajnarwade**
Resolves #80

### CODE REVIEW

File: pkg/cmd/convert.go [hunk 1]
```diff
@@ -138,7 +138,7 @@ func RunConvert(v *viper.Viper, cmd *cobra.Command, out, outerr io.Writer) error
 
 	var writeObject func(o runtime.Object, data []byte) error
 	outputDir := v.GetString(cmdutil.Flag_OutputDir_Key)
-	if outputDir == "-" {
+	if outputDir == "./" {
```
Comment (kadel) [inline pkg/cmd/convert.go]: not sure if this is right solution :-(
It breaks saving output to  current directory.

**Comment by surajnarwade** [comment]
@kadel, 

```
$ opencompose convert -f hello-nginx.yml -o .
created file "helloworld-service.yaml"
created file "helloworld-deployment.yaml"
```

it's working :)

**Comment by surajssd** [comment]
@surajnarwade please update the commit message and title of the PR as well, it is very confusing what change this PR adds.

These might help you:
- https://chris.beams.io/posts/git-commit/
- http://conventionalcommits.org/

**Comment by surajssd** [comment]
I always refer those before I send a PR! :-)

File: pkg/cmd/convert.go (hunk 1 above)
Comment (surajssd) [reply pkg/cmd/convert.go]: This is printing to cmd line even when I give current directory as shown below:
```bash
✔ ~/go/src/github.com/redhat-developer/opencompose/examples/wordpress [pr_155 L|✔] 
12:19 $ opencompose version
v0.2.0+0893f80-clean

✔ ~/go/src/github.com/redhat-developer/opencompose/examples/wordpress [pr_155 L|✔] 
12:19 $ opencompose convert -f storage.yaml -o ./                                                                                                                     
---
apiVersion: v1
kind: Service
metadata:
  creationTimestamp: null
  labels:
...
```

**Review by surajssd — CHANGES_REQUESTED** [review]

**Comment by containscafeine** [comment]
@surajnarwade yep, the commit message is unclear, please include an example in the commit message that what was the previous behavior and what is the new behavior.
Also, we need some tests for this -
- `opencompose convert` prints to STDOUT
- `opencompose convert -o -` throws an error
- `opencompose convert -o .` saves to `pwd`
- `opencompose convert -o ./` saves to `pwd`
- `opencompose convert -o /tmp/` saves to `/tmp/`

**Review by concaf — CHANGES_REQUESTED** [review]
@surajnarwade yep, the commit message is unclear, please include an example in the commit message that what was the previous behavior and what is the new behavior.
Also, we need some tests for this -
- `opencompose convert` prints to STDOUT
- `opencompose convert -o -` throws an error
- `opencompose convert -o .` saves to `pwd`
- `opencompose convert -o ./` saves to `pwd`
- `opencompose convert -o /tmp/` saves to `/tmp/`

**Comment by surajnarwade** [comment]
@kadel @surajssd @containscafeine needs review for this

**Review by kadel — APPROVED** [review]

**Review by concaf — APPROVED** [review]

**Pull request merged**